data-raw/process_data_set_danica.R

```diff
@@ -0,0 +1,64 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#Historical Raptor by Player
+historical_raptor_by_player <- read_csv("data-raw/nba-raptor/historical_RAPTOR_by_player.csv") %>%
+  mutate(
+    era = factor(c("historical"))
+  ) 
+historical_raptor_by_player[c("raptor_box_offense", "raptor_box_defense","raptor_box_total", 
+            "raptor_onoff_offense", "raptor_onoff_defense", "raptor_onoff_total")] <- NA
+historical_raptor_by_player <- historical_raptor_by_player[c("era","player_name","player_id","season","poss","mp","raptor_box_offense","raptor_box_defense","raptor_box_total",    
+                              "raptor_onoff_offense","raptor_onoff_defense","raptor_onoff_total","raptor_offense","raptor_defense","raptor_total",
+                              "war_total","war_reg_season","war_playoffs","predator_offense","predator_defense","predator_total",
+                              "pace_impact")] %>%
+  mutate_all(as.character)
```

Comment:
Don't convert all to `character`. This is making all the numerical variables into characters